xbmc/interfaces/legacy/ListItem.h

```diff
@@ -213,6 +213,58 @@ namespace XBMCAddon
       void setLabel2(const String& label);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_xbmcgui_listitem
+      /// @brief \python_func{ getDateTime() }
+      ///-----------------------------------------------------------------------
+      /// Returns the listitem datetime (ISO 8601).
+      ///
+      /// @return                       datetime (ISO 8601) of item
+      ///
+      ///
+      ///-----------------------------------------------------------------------
+      /// @python_v20 New function added.
+      ///
+      /// **Example:**
+      /// ~~~~~~~~~~~~~{.py}
+      /// ...
+      /// # getDateTime()
+      /// dateTime = listitem.getDateTime()
```

Comment:
I'll adjust the description and the example. Furthermore if I find time I'll take a look at the possibility to return a Python `datetime` object instead of a string.

Reply:
I'd say the conversion is not really important here (just the confusion it can cause). The conversion can be easily done later in the python code itself:

```python
datetime.datetime.strptime(listitem.getDateTime(), "%Y-%m-%dT%H:%M:%S%z")
```

So doing the conversion on the API itself might just add unwanted complexity